lib/executor.js

```diff
@@ -167,6 +168,28 @@ function setupModels(app, instructions) {
   });
 }
 
+function defineMixins(app, instructions) {
+  var modelBuilder = (app.registry || app.loopback).modelBuilder;
+  var BaseClass = app.loopback.Model;
+  var mixins = instructions.mixins || [];
+  if (modelBuilder.mixins && mixins.length > 0) {
+    modelBuilder.mixins.meta = [];
```

Comment:
Let's remove `mixins.meta` for now. AFAIK, it's not used anywhere. We can always introduce it back later.